CreatePackages.md

```diff
@@ -97,6 +97,19 @@ $silentArgs = '/S'
 Install-ChocolateyPackage $packageName $fileType $silentArgs $url
 ```
 
+## During which scenarios will my custom scripts be triggered?
+The table below shows which scripts are available, and which command(s) will cause them to be run.
+
+Script Name                                    | Install | Upgrade | Uninstall
+-----------------------------------------------|---------|---------|----------
+chocolateyInstall.ps1                          | Yes     | Yes     |
+chocolateyBeforeModify.ps1                     |         | Yes     | Yes
```

Comment:
Can you move this to before chocolateyInstall.ps1? That way there is no confusion on the order.